docs/url-formatter.asciidoc

```diff
@@ -0,0 +1,26 @@
+The `Url` field formatter can take on the following types:
+
+* The *Link* type turn the contents of the field into an URL. 
+* The *Image* type can be used to // what?
```

Comment:
Awesome!

I agree that we still need to document the "image" option in the URL formatter session. @spalger is the best person to ask about it. We also need a period at the end of the last sentence in that section :)

Otherwise, LGTM - thanks for working on this!